app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
The directive name can (and you are right should) have that `uc` prefix.
The module name and filenames shouldn't have it. These comments are below the file imports and at some part I probably forgot what the actual directive content looked like.

So `backgroundImage` for the module name, `background.image.module.js` for the module file name and `background.image.directive.js` or something like that where there's no reference to the app name.